Component/DatePicker.js

```diff
@@ -0,0 +1,64 @@
+//Date Picker
+
+import React, { useState } from "react";
+import { Button, Platform, StyleSheet, Switch, Text, View } from "react-native";
+import DateTimePickerModal from "react-native-modal-datetime-picker";
+
+const MainPage = () => {
+  const [pickerMode, setPickerMode] = useState(null);
+  const [inline, setInline] = useState(false);
+
+  const showDatePicker = () => {
+    setPickerMode("date");
+  };
+  
+  const hidePicker = () => {
+    setPickerMode(null);
+  };
+
+  const handleConfirm = (date) => {
+    // In order to prevent the double-shown popup bug on Android, picker has to be hidden first (https://github.com/react-native-datetimepicker/datetimepicker/issues/54#issuecomment-618776550)
+    hidePicker();
+    console.warn("A date has been picked: ", date);
```

Comment:
You can use n ES6 template string for this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
never heard of this let me study